youtube_dl/extractor/tele5.py

```diff
@@ -1,10 +1,13 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import re
+from typing import Optional
```

Comment:
The `typing` module only exists as of Python 3.5.0 but ytdl must work with 2.6.x-2.7.x and 3.2+, if you can believe the [`/setup.py`](https://github.com/ytdl-org/youtube-dl/blob/master/setup.py).